mpqp/execution/providers/google.py

```diff
@@ -310,6 +328,18 @@ def extract_result_STATE_VECTOR(
     return Result(job, state_vector, 0, 0)
 
 
+def extract_result_OBSERVABLE_ideal2(
```

Comment:
why the 2 ?

Reply:
rename to extract_result_OBSERVABLE_processors